spec/policies/exercise_policy_spec.rb

```diff
@@ -48,6 +48,14 @@
     end
   end
 
+  [:show?].each do |action|
```

Comment:
Basically I didn't change application logic but only logic of this test. As Tom told me, I should not change application logic if possible.